backend/business-partner-agent/src/main/resources/databasemigrations/V1.16__add-proof-template-table.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE bpa_proof_template (
+    id uuid PRIMARY KEY,
+    name character varying(255) NOT NULL,
+    attribute_groups jsonb
+);
```

Comment:
If you store the relation to the template in the PartnerProof we also need to add some foreign key constraints here.